lib/puppet/face/module.rb

```diff
@@ -7,7 +7,7 @@
 Puppet::Face.define(:module, '1.0.0') do
   extend Puppet::Util::Colors
 
-  copyright "Puppet Inc.", 2012
+  copyright "Puppet Inc., Vox Pupuli", 2012
```

Comment:
Is the 2012 part correct here, and are the other dates also correct wrt Vox Pupuli?

Reply:
We will have to update the Face implementation to put complex copyrights like that. It supports a single string and a single date. The more accurate copyright adds Vox in 2024, but we can't do that here yet.